mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -1975,7 +1975,7 @@ struct Conv2DRewritePattern : public OpRewritePattern<T> {
     auto hTildaSlice = iHTildaRight - iHTildaLeft;
     auto wTildaSlice = iWTildaRight - iWTildaLeft;
 
-    auto gemmId = 0;
+    auto gemmId = gemmIdAttr.getInt();
```

Comment:
@ltqin  / @asleepzzz please double check if `getGemmA` lambda below carry OOB dimensions. in `getGemmB` and `getGemmC` we set such info but not in `getGemmA`.

Reply:
getGemmA do not have oob check now, please add it

Reply:
still not find out configs need  oob check.

Reply:
embed of wei not need to oob check

Reply:
@asleepzzz please help resolve this conversation if you agree with @ltqin 's analysis.

Reply:
@asleepzzz ping.